src/topp/MSGFPlusAdapter.cpp

```diff
@@ -156,7 +156,7 @@ class MSGFPlusAdapter :
   void registerOptionsAndFlags_() override
   {
     registerInputFile_("in", "<file>", "", "Input file (MS-GF+ parameter '-s')");
-    setValidFormats_("in", ListUtils::create<String>("mzML,mzXML,mgf,ms2"));
+    setValidFormats_("in", {"mzML"});
```

Comment:
this might make some people sad

Reply:
yeah. Maybe.
But getting empty results (no IDs) without knowing why, might make them even more sad (and angry).

Reply:
Also, this makes the Adapter not easily swappable for other adapters, which do not support extra formats.

Reply:
I see the reasons but still think that mzML should be encouraged (maybe even warned if something else is used). This will break existing workflows which will make users even more sad and angry ;)